tests/test_util.py

```diff
@@ -2,11 +2,9 @@
 
 from .helpers import equal_when_sorted
```

Comment:
It doesn't have to be in this PR, but how you feel about restructuring non-flow (unit) tests to match the dir structure of src code? It would mean that we will have a `tests/util` dir that have multiple test files, corresponding to each files in the src `util` package.

Reply:
Yeah, I think that makes sense!